finalPrep/modZeroHero.js

```diff
@@ -2,45 +2,88 @@
 
 // Declare two variables - heroName AND specialAbility - set to strings
 
+var heroName = "Batman";
+var specialAbility = "super smart";
+
 // Declare two variables - greeting AND catchphrase
 //   greeting should be assigned to a string that uses concatenation to include the heroName
 //   catchphrase should be assigned to a string that uses interpolation to include the specialAbility
 
+var greeting = "Hello " + heroName + " , welcome home."
+var catchphrase = "Not to fear citizens, I'm ${specialAbility}!"
+
 // Declare two variables - power AND energy - set to integers
 
+var power = 10
+var energy = 20
+
 // Declare two variables - fullPower AND fullEnergy
 //   fullPower should multiply your current power by 500
 //   fullEnergy should add 150 to your current energy
 
+var fullPower = power * 500
+var fullEnergy = energy + 150
+
 // Declare two variables - isHuman and identityConcealed - assigned to booleans
 
+var isHuman = false
+var identityConcealed = true
 
 // Declare two variables - archEnemies AND sidekicks
 //   archEnemies should be an array of at least 3 different enemy strings
 //   sidekicks should be an array of at least 3 different sidekick strings
 
+var archEnemies = ["Joker", "Harley-Quinn", "Riddler"];
+var sidekicks = ["Robin", "Commissioner Gordon", "Catwoman"];
+
 // Print the first sidekick to your console
 
+console.log(sidekicks[0]);
+
 // Print the last archEnemy to the console
 
+console.log(archEnemies[2]);
```

Comment:
What is an alternative way to print the last `archEnemy` to the console, particularly if you didn't know the length of the array?

Reply:
Hi, @mschae16! 

Another way to print the last `archEnemy` to the console would be to instead use the length property of the archEnemies array minus one (since the last element in the array is one less than the total number of array elements):
```Javascript
console.log(archEnemies[archEnemies.length - 1]);
```
This would then always print the last element even if the array length changes!

Reply:
Nice one! 💯